iohub/ngff/nodes.py

```diff
@@ -1010,6 +1010,43 @@ def get_axis_index(self, axis_name: str) -> int:
         """
         return self.axis_names.index(axis_name.lower())
 
+    def get_transforms(
+        self,
+        image: str | Literal["*"],
+    ) -> tuple[TransformationMeta, TransformationMeta]:
+        """Get the total coordinate scale and translation metadata
+        for one image array or the whole FOV.
+
+        Parameters
+        ----------
+        image : str | Literal["*"]
+            Name of one image array (e.g. "0") to transform,
+            or "*" for the whole FOV
+        """
+        transforms: list[TransformationMeta] = self.metadata.multiscales[0].coordinate_transformations
+        if image != "*" and image in self:
+            for i, dataset_meta in enumerate(self.metadata.multiscales[0].datasets):
+                if dataset_meta.path == image:
+                    transforms.extend(
+                        self.metadata.multiscales[0]
+                        .datasets[i]
+                        .coordinate_transformations
+                    )
+        elif image != "*":
+            raise ValueError(f"Key {image} not recognized.")
+
+        full_scale = np.array([1]*len(self.axes), dtype=float)
+        full_translation = np.array([0]*len(self.axes), dtype=float)
```

Comment:
Likely inconsequential but vectorized:

```suggestion
        full_scale = np.ones(len(self.axes), dtype=float)
        full_translation = np.zeros(len(self.axes), dtype=float)
```